tools/SetupFlow/DevHome.SetupFlow/Views/ConfigurationFileView.xaml

```diff
@@ -17,8 +17,10 @@
     </setupFlowBehaviors:SetupFlowNavigationBehavior.ContentTemplate>
     <setupFlowBehaviors:SetupFlowNavigationBehavior.NextTemplate>
         <ToggleSplitButton
+            x:Name="SetUpButton"
             IsEnabled="{x:Bind ViewModel.ReadAndAgree, Mode=OneWay}"
-            IsChecked="{x:Bind ViewModel.ReadAndAgree, Mode=OneWay}"
+            IsEnabledChanged="SetUpButton_IsEnabledChanged"
+            IsCheckedChanged="SetUpButton_IsCheckedChanged"

```

Comment:
Similar to a checkbox, a "checked" toggle button means that it's "activated". Visually, a checked button would appear similar to a primary button (with accent color), and when unchecked, it would look like a default button.